doc/nrf-bm/libraries/nfc/index.rst

```diff
@@ -0,0 +1,75 @@
+.. _lib_nfc:
+
+NFC libraries
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The NFC libraries support NFC-A listen mode, where the tag waits for polling devices but does not actively start a connection.
+This makes them ideal for implementing NFC tag functionality in bare metal applications.
+
+Supported NFC tag types
+***********************
+
+The following NFC tag types are supported:
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Tag Type
+     - Description
+     - Library
+   * - Type 2 Tag (T2T)
+     - Read-only NDEF tag with 992 bytes of data
+     - :ref:`lib_nfc_t2t`
+   * - Type 4 Tag (T4T)
+     - Read-only or read-write NDEF tag with up to 64 KB of data
+     - :ref:`lib_nfc_t4t`
+
+Integration requirements
+************************
+
+When integrating the NFC libraries in your application, be aware of the following:
+
+* The libraries require the NFCT driver from the `nrfx`_ repository.
+* The NFCT driver uses one Timer peripheral.
+  You can configure the timer instance by changing ``NRFX_NFCT_CONFIG_TIMER_INSTANCE_ID`` in the :file:`nrfx_config.h` header.
+* Each library must be the only user of the NFCT peripheral and the chosen TIMER instance.
+* To use the libraries, you must implement an NFC Platform software module that handles clock control and interrupt routing.
```

Comment:
This is implemented in the BM repo (subsys/nfc/lib/nfc_bm_platform.c). The documentation should be written to reflect that, and not that this is something the user must implement. Same comment applies for the section below.

Reply:
Right, I changed these sentences.